OPEN_PROBLEMS/DECENTRALIZED_DATA_DELIVERY_MARKETS.md

```diff
@@ -0,0 +1,509 @@
+# Decentralized Data Delivery Markets (3DMs) - Open Problem Statement
+
+DO NOT MERGE WITHOUT CREATING DOCTOC https://www.npmjs.com/package/doctoc
+
+# Overall
+
+## Short description
+
+With the emergence of Decentralized Storage Networks and the rapid decrease in the price of storage services and hardware, there is a rapidly growing need to leverage the additional storage capacity contributed to Decentralized Storage Networks by several new players, such as end-users and use it to deliver a reliable and high quality storage but also delivery services. Similarly to Content Delivery Networks (CDNs) for the traditional Cloud Storage market, we now have the opportunity to build **Decentralised CDNs **for Decentralised Storage Networks. The Decentralized Data Delivery Markets (3DMs) Open Problem covers all the essential areas of work that need to be studied in order to create **a fully permissionless free market for data delivery that supports fair data exchange** on the service provided.
+
+## Long description 
+
+Serving content globally at scale is a hard technical problem as evidenced by the multiple decades of innovation and improvements in the Content Delivery Networks field. This challenge becomes even more interesting once we move away from centralized cloud infrastructures, which is typically managed and monitored by a single party, to a decentralized network that is permissionless, likely anonymous, constantly changing (i.e. high node churn) and without access to the convenience of a third party mediator system that facilitates the fair exchange of goods (e.g. credit providers). The benefits of moving to a decentralised setting are significant, however: cheaper storage and delivery services, resilience against business failures (i.e., the network and all its components are not dependent on business decisions made by a single entity), moving away from the personal data-based business models, as well as a significantly lower barrier to entry for new players who want to contribute to the system.
+
+At the same time, common problems addressed in the past, in traditional CDNs, reappear, albeit in a different dimension as we move to a decentralized network design space. Some of these problems are: the global orchestration of the system, the efficient allocation and use of the resources, or a clear visibility of the content being retrieved from the network.
```

Comment:
```suggestion
At the same time, problems already addressed in the past for traditional CDNs reappear, albeit in a different dimension, as we move to a decentralized network design space. These problems include the global orchestration of the system, the efficient allocation and use of the resources, and clear visibility of the content being retrieved from the network.
```

Reply:
The phrasing here is a little unclear: do we consider the content being visible a problem (i.e. is this a privacy issue?) or do we consider the lack of visibility a problem (i.e. is this a network optimisation issue?).